src/modules/feature-modules/innovator/pages/innovation/export-requests/export-request-reject.component.html

```diff
@@ -0,0 +1,14 @@
+<theme-content-wrapper [status]="pageStatus">
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <form [formGroup]="form" (ngSubmit)="onSubmit()">
+        <theme-form-textarea controlName="rejectReason" label="Reject request" description="Explain the reason for rejecting this request" [pageUniqueField]="true" lengthLimit="s"></theme-form-textarea>
```

Comment:
Missing full stop in the description: `Explain the reason for rejecting this request.`

Reply:
Done. Thanks.